functions/index.js

```
@@ -0,0 +1,68 @@
+// Callable to unsubscribe a token from multiple topics
+export const unsubscribeFromTopics = functions.https.onCall(async (data, context) => {
+  const { token, topics } = data || {};
+  if (!token || !Array.isArray(topics)) return { ok: false };
+  for (const topic of topics) {
+    try { await admin.messaging().unsubscribeFromTopic(token, topic); } catch (e) { /* no-op */ }
+  }
+  return { ok: true };
+});
+
+// Manual push trigger for testing
+export const sendCommunityPushManual = functions.https.onCall(async (data, context) => {
```

ESLint

[error] 'context' is defined but never used.

(no-unused-vars)